File: 工程文件/服务器端/爬虫/CMD.py

AppMsg={
    'cmd':0,
    'ReqHotList':{'wechat_id_list':[]},
    'RspHotList':{'article_cover_list':[]},
    'ReqSearchList':{'input_text':''},
    'RspSearchList':{
        'RspSearchList':[],
        'offcial_account_list':[]
    },
    'ReqArticleContent':{
        'title':'',
        'author':''
    },
    'RspArticleContent':{
        'Article':{
                    #文章题目
                    'article_id':'',
                    #文章地址
                    'article_url':'',
                    #文章图片
                    'article_imgs':'',
                    #文章摘要
                    'article_abstract':'',
                    #文章来源公众号头像
                    'gzh_headimage':'',
                    #文章来源公众号id
                    'gzh_id':'',
        }
    },
    'ReqOAInfo':{'wechat_id':''},
    'RspOAInfo':{
        'OffcialAccount':{
                'open_id': '', # 微信号唯一ID
                'headimage': '',  # 头像
                'wechat_name': '',  # 名称
                'wechat_id': '',  # 微信id
                'post_perm': '',  # 最近一月群发数
                'introduction': '',  # 介绍
        }
    },
    'RcvQRCode':{'qrcode':[]},
    'BackQRCode':{'qrcodestring':''},

}

File: 工程文件/客户端/WCH/Assets/Script/Net/CMD.cs

using System.Collections.Generic;
using UnityEngine;


namespace WCH
{  
    public class AppMsg {

        public int cmd; //请求类型

        public ReqHotList reqHotList = new ReqHotList();
        public RspHotList rspHotList = new RspHotList();
        public ReqSearchList reqSearchList = new ReqSearchList();
        public RspSearchList rspSearchList = new RspSearchList();
        public ReqArticleContent reqArticleContent = new ReqArticleContent();
        public RspArticleContent rspArticleContent = new RspArticleContent();
        public ReqOAInfo reqOAInfo = new ReqOAInfo();
        public RspOAInfo rspOAInfo = new RspOAInfo();

        public RcvQRCode rcvQRCode = new RcvQRCode();
        public BackQRCode backQRCode = new BackQRCode();
    }

    #region 请求数据
    public class ReqHotList {
        public List<string> wechat_id_list = new List<string>();//发送关注名单
    }
    public class RspHotList {
        public List<ArticleCover> article_cover_list = new List<ArticleCover>(); //返回文章封面信息列表
    }
    public class ReqSearchList {
        public string input_text="";//搜索文字
    }
    public class RspSearchList {
        public List<ArticleCover> article_cover_list = new List<ArticleCover>(); //返回文章封面信息列表
        public List<OffcialAccount> offcial_account_list = new List<OffcialAccount>(); //返回公众号封面信息列表
    }
    public class ReqArticleContent {
        public string title="";//文章标题 
        public string author=""; //作者
    }
    public class RspArticleContent {
        public Article article = new Article();
    }
    public class ReqOAInfo
    {
        public string wechat_id="";//微信ID
    }
    public class RspOAInfo
    {
        public OffcialAccount offcialAccount = new OffcialAccount(); 
    }
    public class RcvQRCode
    {
        public string qrcode = "";
    }
    public class BackQRCode
    {
        public string qrcodestring="";
    }

    #endregion


    public enum CMD
    {
        HOTARTICLECOVERINFO = 0,//获取热门文章列表
        SEARCHRESULT = 1,//搜索结果
        ARTICLECONTENT = 2,//文章内容
        ARTICLECOVERANDOA = 3,//主页获取 
        ORCODE = 4,//二维码
    }




}

